modules/test/playwright/pages/portlet-configuration-web/PortletConfigurationPermissionsPage.ts

```diff
@@ -0,0 +1,53 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+// @ts-ignore
+
+import {FrameLocator, Locator, Page, expect} from '@playwright/test';
+
+import {ProductMenuPage} from '../product-navigation-control-menu-web/ProductMenuPage';
+
+export class PortletConfigurationPermissionsPage {
+    readonly clearLink : Locator;
+    readonly ownerRoleCell : Locator;
+    readonly page: Page;
+    readonly pageOptionsMenu : Locator;
+    readonly permissionsFrame : FrameLocator;
+    readonly permissionsMenuItem : Locator;
+    readonly productMenuPage : ProductMenuPage;
+    readonly resultsBanner : Locator;
+    readonly saveButton: Locator;
+    readonly searchBar : Locator;
+    readonly siteMemberRoleCell : Locator;
+    readonly successMessage : Locator;
+
+    constructor(page : Page) {
+        this.page = page;
+        this.pageOptionsMenu = page.getByTitle("Open Page Options Menu");
+        this.permissionsFrame = page.frameLocator('iframe[title="Permissions"]');
+        this.permissionsMenuItem = page.getByRole('menuitem', { name: 'Permissions' });
+        this.productMenuPage = new ProductMenuPage(page);
+
+        this.clearLink = this.permissionsFrame.getByLabel('Clear');
+        this.ownerRoleCell = this.permissionsFrame.getByRole('cell', {name : 'Owner'});
+        this.resultsBanner = this.permissionsFrame.getByText('Results Found for');
+        this.saveButton = this.permissionsFrame.getByRole('button', { name: 'Save' });
+        this.searchBar = this.permissionsFrame.getByPlaceholder('Search for');
+        this.siteMemberRoleCell = this.permissionsFrame.getByRole('cell', {name : 'Site Member'});
+        this.successMessage = this.permissionsFrame.getByText('Success:Your request completed successfully.');
```

Comment:
I think this can be replaced by an utility already available to check successMessage (search for `waitForSuccessAlert`)

Reply:
I actually tried to use this previously but wasn't able to pass in the frame successfully (so it would never find the success alert). I'll try again, but I'm not sure it would work in this case.